diff --git a/test/e2e/helpers/kafka_source_test_helper.go b/test/e2e/helpers/kafka_source_test_helper.go
index a8976e593..ea8e05484 100644
--- a/test/e2e/helpers/kafka_source_test_helper.go
+++ b/test/e2e/helpers/kafka_source_test_helper.go
@@ -248,11 +248,11 @@ func TestKafkaSourceUpdate(t *testing.T) {
 			topicName: defaultKafkaSource.topicName,
 			sinkName:  "update-event-recorder",
 		},
-		"change-topic": {
-			auth:      defaultKafkaSource.auth,
-			topicName: "update-topic",
-			sinkName:  defaultKafkaSource.sinkName,
-		},
+		// "change-topic": {
+		// 	auth:      defaultKafkaSource.auth,
+		// 	topicName: "update-topic",
+		// 	sinkName:  defaultKafkaSource.sinkName,
+		// },
 		"change-bootstrap-server": {
 			auth: authSetup{
 				bootStrapServer: KafkaBootstrapUrlPlain,
