diff --git a/pkg/common/client/config.go b/pkg/common/client/config.go
index d96ea86eb..cca8447d8 100644
--- a/pkg/common/client/config.go
+++ b/pkg/common/client/config.go
@@ -198,6 +198,7 @@ func (b *configBuilder) Build(ctx context.Context) (*sarama.Config, error) {
 
 		// We Always Want Success Messages From Producer
 		config.Producer.Return.Successes = true
+		config.Producer.RequiredAcks = sarama.WaitForAll
 	}
 
 	// then apply stuff from YAML
